tests/unit/test_models/test_full_battery_models/test_lithium_ion/test_newman_tobias.py

```diff
@@ -1,42 +1,37 @@
 #
 # Tests for the lithium-ion Newman-Tobias model
 #
-from tests import TestCase
 import pybamm
-import unittest
+import pytest
 from tests import BaseUnitTestLithiumIon
 
 
-class TestNewmanTobias(BaseUnitTestLithiumIon, TestCase):
+class TestNewmanTobias(BaseUnitTestLithiumIon):
+    @pytest.fixture(autouse=True)
     def setUp(self):
         self.model = pybamm.lithium_ion.NewmanTobias
 
     def test_electrolyte_options(self):
         options = {"electrolyte conductivity": "integrated"}
-        with self.assertRaisesRegex(pybamm.OptionError, "electrolyte conductivity"):
+        with pytest.raises(pybamm.OptionError, match="electrolyte conductivity"):
             pybamm.lithium_ion.NewmanTobias(options)
 
+    @pytest.mark.skip
```

Comment:
I've added skips but logs show "unconditional skip". Should we add a reason here or that's fine?

Reply:
Something like "Test currently not implemented" would be great, it serves additional context in comparison to "Unconditional skip"